src/utils/blockchainHelpers.js

```diff
@@ -213,49 +213,17 @@ export function deployContract(i, web3, abi, bin, params, state, cb) {
   });
 }
 
-export function sendTXToContract(web3, method, cb) {
-  let isMined = false;
+export function sendTXToContract(method, cb) {
   method
-  .on('error', function(error) {
-    console.log(error);
-    return cb(error);
-  })
-  .on('transactionHash', function(transactionHash){
-    console.log("contract method transaction: " + transactionHash);
-
-    checkTxMined(web3, transactionHash, function txMinedCallback(receipt) {
```

Comment:
This additional polling of tx receipt was made, because users had problems on mainnet: wizard hanged on random transaction, because there wasn't response from it, no receipt. Especially, if you switch between tabs when wizard works. Did byzantium hard fork fix it? I doubt, that we need to change this part without real need, even if hand tests will successfull on mainnet.

Reply:
Ok, good to know. I'll rollback the changes, and will leave a comment with that explanation for future references. Thanks for the input!